service/src/test/java/bio/terra/profile/service/profile/ProfileServiceUnitTest.java

```diff
@@ -74,18 +74,18 @@ public void before() {
   @Test
```

Comment:
Looking for feedback on test coverage for this change. Do you have any suggestions?

Reply:
This seems like a reasonable to the test given your changes above.